packages/character-count-ui/package.json

```diff
@@ -30,15 +30,15 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "slate": "^0.60.0",
-    "slate-react": "^0.60.0"
+    "slate-react": "^0.60.0",
+    "lodash": "^4.17.21"
```

Comment:
and i think this should also not be in the `peerDependencies`

Reply:
added it to dependencies